r-pkg/uptasticsearch/docs/reference/unpack_nested_data.html

```diff
@@ -123,7 +123,7 @@ <h2 class="hasAnchor" id="examples"><a class="anchor" href="#examples"></a>Examp
   {"film":"Rogue One","pmt_amount":10.25},{"film":"Bridesmaids",
   "pmt_amount":8.75},{"film":"Bridesmaids","pmt_amount":6.25, "matinee": 
   true}]}}}]'</span>
-  , <span class='kw'>keep_nested_data_cols</span> <span class='kw'>=</span> <span class='fl'>TRUE</span>)</div><div class='output co'>#&gt; INFO [2017-07-14 15:03:41] Keeping the following nested data columns. Consider using unpack_nested_data for one:
+  , <span class='kw'>keep_nested_data_cols</span> <span class='kw'>=</span> <span class='fl'>TRUE</span>)</div><div class='output co'>#&gt; INFO [2017-07-15 21:57:24] Keeping the following nested data columns. Consider using unpack_nested_data for one:
```

Comment:
Oh this is wild: pkgdown automatically runs examples that aren't wrapped in `/dontrun` and displays the output in the reference page. Crazy!